### DISCUSSION TITLE
REST Auth user id/email

### DISCUSSION THREAD

**Discussion by tkowalski29** [Budibase/budibase#4585] [Help] [answered]
Hi, how can i add my user id or user email to header in rest?

For example i want add BUDIBASE-User-id or BUDIBASE-User-email. 

I need this to verify which user is currently using api

**Comment by mjashanks** [comment, marked as answer]
Hey @tkowalski29 - unfortunately this is not possible right now. We have an issue for this: https://github.com/Budibase/budibase/issues/4139

We definitely want to get this one done soon, but not sure when it will happen.

**Comment by tkowalski29** [comment]
It is a pity, but I am looking forward to moving my entire panel.
Thank you for the information
